src/main/java/org/mule/extension/http/api/request/builder/SimpleRequestHeader.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.http.api.request.builder;
+
+import org.mule.runtime.extension.api.annotation.Alias;
+import org.mule.runtime.extension.api.annotation.param.Parameter;
+
+import java.util.Objects;
+
+/**
+ * Represents an HTTP Header
+ *
+ * @since 1.7
+ */
+@Alias("requestHeader")
+public class SimpleRequestHeader implements KeyValuePair {
```

Comment:
why is the class called `SimpleRequestHeader` instead of simply `RequestHeader`?

Reply:
Because there is already a RequestHeader (which is used somewhere else and we can't reuse here), in the same Package. Having a package for this to then have the same name would be quite misleading